spec/rails_helper.rb

```diff
@@ -0,0 +1,27 @@
+ENV['RAILS_ENV'] ||= 'test'
+require File.expand_path('../config/environment', __dir__)
+
+abort('The Rails environment is running in production mode!') if Rails.env.production?
+
+require 'rspec/rails'
+require 'spec_helper'
+# require 'webmock/rspec'
```

Comment:
what do you think about moving the webmock stuff to a separate file in `/support`?